SPA/views/pages/Interviewers.js

```diff
@@ -0,0 +1,112 @@
+import Redirect        from '../../services/Redirect.js'
+import getInterviewersList        from '../../services/getInterviewersList.js'
+
+
+  window.deleteInterviewer = async (id) => {
+  
+    const confirm = window.confirm("Are you sure? ");
+    if(confirm)
+    {
+         const options = {
+             method: 'DELETE',
+             headers: {
+              'Content-Type': 'application/json',
+              
+             }
+         };
```

Comment:
Why did we need to bind this function with window object? Also, since this function is in use too many times, it can be moved to Utils and directly be exported from there.

Reply:
Without binding this deleteInterviewer function to the window object I am facing an error that this particular function is not defined.